crates/shim-kvm/src/syscall.rs

```diff
@@ -135,6 +135,30 @@ extern "sysv64" fn syscall_rust(
     let usermemscope = UserMemScope;
 
     match nr as i64 {
+        SYS_GETKEY => {
+            let ret = h.get_key(&usermemscope, a, b);
+
+            match ret {
+                Err(e) => {
+                    #[cfg(feature = "dbg")]
+                    eprintln!("syscall {} = {}", nr, e.checked_neg().unwrap());
+                    X8664DoubleReturn {
+                        rax: e.checked_neg().unwrap() as _,
+                        // Preserve `rdx` as it is normally not clobbered with a syscall
+                        rdx: orig_rdx as _,
+                    }
+                }
+                Ok(rax) => {
+                    #[cfg(feature = "dbg")]
+                    eprintln!("syscall {} = {}", nr, rax);
```

Comment:
Would it be better to combine these two logging statements into one above the match?

Reply:
done